### PR TITLE
Update the libraries that work with OkHttp

### PR DESCRIPTION
**Pull request by swankjesse**
This prunes a few that haven't been updated and appear obsolete.

### CODE REVIEW

File: docs/works_with_okhttp.md [hunk 1]
```diff
@@ -3,14 +3,14 @@ Works with OkHttp
 
 Here’s some libraries that work nicely with OkHttp.
 
- * [Chuck](https://github.com/jgilfelt/chuck): An in-app HTTP inspector for Android OkHttp clients.
+ * [Chucker](https://github.com/ChuckerTeam/chucker): An in-app HTTP inspector for Android OkHttp clients.
+ * [Coil](https://github.com/colinrtwhite/coil): An image loading library for Android backed by Kotlin Coroutines.
```
Comment (monkey-mas) [inline docs/works_with_okhttp.md]: Seems like they moved to `https://github.com/coil-kt/coil`? 🙄

**Comment by yschimke** [comment]
https://fbflipper.com/ ?

File: docs/works_with_okhttp.md (hunk 1 above)
Comment (colinrtwhite) [reply docs/works_with_okhttp.md]: Yep! `https://github.com/coil-kt/coil` is the correct URL.

Comment (swankjesse) [reply docs/works_with_okhttp.md]: Fixed.

**Comment by swankjesse** [comment]
Added Flipper.

**Pull request merged**